### PR TITLE
[BLCR-3248] Add documentation for 401 expired session response

### PR DESCRIPTION
**Pull request by Netznarkose**
[Ticket](https://brandslisten.atlassian.net/browse/BLCR-3248)

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -195,6 +195,14 @@ paths:
           description: Successful response
           schema:
             $ref: '#/definitions/User'
+        '401':
+          description: Not Authorized
```
Comment (Xtimesx) [inline swagger.yaml]: can you mention that the X-User-Token is not valid anymore/ at all in this case?
just so other developers know what is probably wrong

Comment (Netznarkose) [reply swagger.yaml]: sure

Comment (Netznarkose) [reply swagger.yaml]: if it looks good to you, I apply it to the other related endpoints

Comment (erplsf) [reply swagger.yaml]: should we add it to all endpoints? also, the message you return in error is 'expired session' but here you say "Not Authorized" - why not write something that matches endpoint here (401 not authorized is http code, but we can ship different messages with them)

Comment (Netznarkose) [reply swagger.yaml]: @erplsf  I wrote `to the other related endpoints` so endpoints that require the x-user-token.
you're right this way the response is pretty inconsistent I'll fix that.

Comment (Netznarkose) [reply swagger.yaml]: @erplsf can you please check again?

**Pull request merged**